### DISCUSSION TITLE
Parameter ignore_volume_control = "yes" should disable volume control on Airplay sender

### DISCUSSION THREAD

**Discussion by cs-hv** [mikebrady/shairport-sync#1409] [Ideas]
In my shairport-sync profile I have set **ignore_volume_control** to "yes" because I use a DAC with an external amplifier. This configuration is working as expected however everytime I choose my shairport-device as audio output on my Mac / iOS device, the volume control shows to be around 1/3 (I think it's -20dB).
It would be clearer, if the volume control would initially show 100% or if it would even be disabled when ignore_volume_control is set to yes. From what I found when searching the internet, there are Apple TV configurations where Airplay volume control is disabled. I assume that the Airplay receiver can somehow tell the sender hat volume control has to be disabled. I would like to suggest that shairport-sync behaves the same way under the condition mentioned above.

**Comment by mikebrady** [comment]
Thanks for the suggestion -- I'll have a think about it.

**Comment by mikebrady** [comment]
I thought some more about this -- and even implemented some code -- but it's a non-runner because while the volume control would be ignored by Shairport Sync, it is often used by external volume control systems. In those situations, as you can imagine, setting it to 100% automatically would be a Really Bad Idea™ 😊.

Thanks for the suggestion though -- it's always good to get new ideas.

**Reply by noelhibbard** [reply]
>  In those situations, as you can imagine, setting it to 100% automatically would be a Really Bad Idea™ 😊.

Yep, this change would have had some bad side effects in my setup. I have one of my instances connected to an AVR but have shairport-sync setup to control my AVR volume.
